TestAttachStripped failure with undo backend

```
$ go run scripts/make.go test -v -s proc -r TestAttachStripped -b undo
=== RUN   TestAttachStripped
--- FAIL: TestAttachStripped (0.43s)
    proc_test.go:3103: unknown backend "undo"
```
For now we could handle this in the same was as `"rr"`, that is, pass the test without attaching.

There is a similar case in `TestAttachDetach` which we could handle in the same way.

--- a/pkg/proc/proc_test.go
+++ b/pkg/proc/proc_test.go
@@ -2759,7 +2759,7 @@ func TestAttachDetach(t *testing.T) {
 			return
 		}
 	}
-	if testBackend == "rr" {
+	if testBackend == "rr" || testBackend == "undo" {
 		return
 	}
 	var buildFlags protest.BuildFlags
@@ -3057,7 +3057,7 @@ func TestAttachStripped(t *testing.T) {
 			return
 		}
 	}
-	if testBackend == "rr" {
+	if testBackend == "rr" || testBackend == "undo" {
 		return
 	}
 	if runtime.GOOS == "darwin" {

